### PR TITLE
Feature/db

### PR DESCRIPTION
**Pull request by chumaknadya**

### CODE REVIEW

File: migrations/sqls/20220803122605-create-tables-up.sql [hunk 1]
```diff
@@ -1,34 +1,29 @@
...
-    flag_href text
-);
-
-CREATE TABLE public.emails (
-    address text NOT NULL,
-    email text NOT NULL
+    uid uuid UNIQUE NOT NULL,
+    state varchar(32) NOT NULL DEFAULT 'active',
+    data jsonb,
+    referral_id bigint,
```
Comment (mod) [inline migrations/sqls/20220803122605-create-tables-up.sql:6]: name it ref_id move right after ID ( I always place indexed and FK field at beginning of table)

File: migrations/sqls/20220803122605-create-tables-up.sql [hunk 2]
```diff
@@ -1,34 +1,29 @@
...
+    created_at timestamp(0) DEFAULT now(),
```
Comment (mod) [inline migrations/sqls/20220803122605-create-tables-up.sql:7]: no ability to have updated_at in supabase or so ?

File: migrations/sqls/20220803122605-create-tables-up.sql [hunk 3]
```diff
@@ -1,34 +1,29 @@
...
 );
 
-CREATE TABLE public.rewards (
-    address text NOT NULL,
-    last_daily_received_at timestamp without time zone,
-    daily_streak_count smallint DEFAULT '0'::smallint
+CREATE TABLE public.phones (
+    id bigint NOT NULL,
+    user_id bigint,
+    number varchar(128) UNIQUE NOT NULL,
```
Comment (mod) [inline migrations/sqls/20220803122605-create-tables-up.sql:23]: I suggest country code if possible, can be useful for searches
Also do we make totp or random otp
if random numbers we need to save them unfortunately

Comment (mod) [reply migrations/sqls/20220803122605-create-tables-up.sql:23]: unless HMACed ?

**Review by mod — CHANGES_REQUESTED** [review]

File: migrations/sqls/20220803122605-create-tables-up.sql (hunk 1 above)
Comment (chumaknadya) [reply migrations/sqls/20220803122605-create-tables-up.sql:6]: ok

File: migrations/sqls/20220803122605-create-tables-up.sql (hunk 2 above)
Comment (chumaknadya) [reply migrations/sqls/20220803122605-create-tables-up.sql:7]: for sure, we can do, lets do that as convention from all projects

File: migrations/sqls/20220803122605-create-tables-up.sql (hunk 3 above)
Comment (chumaknadya) [reply migrations/sqls/20220803122605-create-tables-up.sql:23]: I would like to use totp

**Pull request closed without merging**